editor/src/clj/editor/properties_view.clj

```diff
@@ -290,9 +290,14 @@
                         (ui/editable! color-picker (not read-only?)))]
 
     (ui/on-action! color-picker (fn [_] (let [^Color c (.getValue color-picker)
-                                              v        [(.getRed c) (.getGreen c) (.getBlue c) (.getOpacity c)]
+                                              v        [(math/round-with-precision (.getRed c) 0.001)
+                                                        (math/round-with-precision (.getGreen c) 0.001)
+                                                        (math/round-with-precision (.getBlue c) 0.001)
+                                                        (math/round-with-precision (.getOpacity c) 0.001)]
```

Comment:
We now round the double color component values we get from the color picker to a sane precision.